README.md

```diff
@@ -21,6 +22,7 @@ npm install --save fetch-har
 ```
 
 ## Usage
+
 ```js
 import fetchHAR from 'fetch-har';
 // const fetchHAR = require('fetch-har').default;
```

Comment:
Based on this conversation: https://github.com/readmeio/fetch-har/pull/427/files#r1337662183

```suggestion
// const fetchHAR = require('fetch-har');
```

Reply:
oh great catch, thank you!